### PR TITLE
Tos7 po box

### PR DESCRIPTION
**Pull request by jonescc**
Add TOS 7 PO Box

### CODE REVIEW

**Comment by jonescc** [comment]
Requires talend-spatial/talend-spatial#78 or other fiix for talend-spatial/talend-spatial#77

**Comment by jonescc** [comment]
Ready for review - will get an autosave error running tos 7.1.1 every 5 mins until talend spatial issue above is resolved or we'll need to use local desktop/docker installs of TOS instead.

File: bin/vagrant-wrapper.sh [hunk 1]
```diff
@@ -218,11 +227,64 @@ talend_main() {
...
+    _run_vm $PO7_VM_NAME "$@" || return 1
+}
+
+# main function for running talend
+talend7_main() {
+    declare -r TALEND_CUSTOM_COMPONENTS="src/talend-components-7"
+    declare -r TALEND_COMPONENTS_DOWNLOAD_URL="http://imos-binary.s3-website-ap-southeast-2.amazonaws.com/jobs/adhoc_talend_components_build/adhoc_talend_components_build-latest.zip"
+    declare -r HARVESTERS_GIT_REPO="git@github.com:aodn/harvesters.git"
+
+    _clone_git_repo $HARVESTERS_GIT_REPO src/harvesters-7 tos-7-1-1 || return 1
```
Comment (gsatimos) [inline bin/vagrant-wrapper.sh]: guess we need to note that when we do the talend 7 clean-up next iteration, that we 
1. merge tos-7-1-1 branch into master on harvesters repo
2. update the this line from tos-7-1-1 to master
3. then the tos-7-1-1 branch in harvesters repo can be deleted
4. Update TALEND_COMPONENTS_DOWNLOAD_URL after jenkins job talend_components_edge is updated

sounds right @jonescc ?

sounds right @

Comment (jonescc) [reply bin/vagrant-wrapper.sh]: sounds right @gsatimos

**Comment by gsatimos** [comment]
you were going to update the main readme with a link to the specific tos7 readme?

**Comment by jonescc** [comment]
Added link to TOS 7 PO Box documentation to main README

**Pull request merged**